modules/apps/frontend-js/frontend-js-svg4everybody-web/src/main/java/com/liferay/frontend/js/svg4everybody/web/internal/servlet/taglib/SVG4EverybodyTopHeadDynamicInclude.java

```diff
@@ -48,7 +51,24 @@ public void include(
 			HttpServletResponse httpServletResponse, String key)
 		throws IOException {
 
-		if (!PropsValues.CDN_DYNAMIC_RESOURCES_ENABLED ||
+		boolean cdnDynamicResourcesEnabled;
+
+		try {
+			cdnDynamicResourcesEnabled = _portal.isCDNDynamicResourcesEnabled(
```

Comment:
@ChrisKian Isn't the Portal.isCDNDynamicResourcesEnabled() already fallback to PropsValues.CDN_DYNAMIC_RESOURCES_ENABLED ? Why do you need to do this explicit fallback?